keyboards/pk60/pk60.c

```diff
@@ -7,6 +7,5 @@ void led_set_kb(uint8_t usb_led) {
     } else {
         PORTF &= ~(1<<4);
     }
-    led_set_user(usb_led);
```

Comment:
This will remove the ability for the function to be called in the keymap. 

Also, there may be somer initialization that needs to be done in `matrix_init_kb` to get the LED working

Reply:
Specifically, you may need: 

```
void matrix_init_kb (void) {
  DDRF |= (1 << 4);
  PORTF &= ~(1 << 4);
  matrix_init_user();
}
```
Try adding this to your keymap, and see if that works locally.